src/platform/Linux/PlatformManagerImpl.cpp

```diff
@@ -124,6 +178,14 @@ void PlatformManagerImpl::WiFIIPChangeListener()
 CHIP_ERROR PlatformManagerImpl::_InitChipStack()
 {
     CHIP_ERROR err;
+    struct sigaction action;
+
+    memset(&action, 0, sizeof(action));
+    action.sa_handler = SignalHandler;
+    sigaction(SIGTERM, &action, NULL);
+    sigaction(SIGINT, &action, NULL);
+    sigaction(SIGHUP, &action, NULL);
+    sigaction(SIGKILL, &action, NULL);
```

Comment:
SignalHandler looks for other signals too, no?

Reply:
I use signal to simulate different reboot reason on Linux, only six signals are looked for, no other signals.